storage_service/locations/models/wellcome.py

```diff
@@ -29,23 +34,28 @@ def handle_ingest(ingest, package):
     """
     status = ingest['status']['id']
     if status == 'succeeded':
-        external_id = ingest['bag']['info']['externalIdentifier']
         package.status = Package.UPLOADED
-        package.misc_attributes['ingest_id'] = ingest['id']
-        package.current_path = external_id
+        # Strip the directory context from the package path so it is
+        # in the format NAME-uuid.tar.gz
+        package.current_path = os.path.basename(package.current_path)
+        bag_info = ingest['bag']['info']
+        package.misc_attributes['bag_id'] = bag_info['externalIdentifier']
+        package.misc_attributes['bag_version'] = bag_info['version']
+
+        LOGGER.debug('Package path: %s', package.current_path)
+        LOGGER.debug('Package attributes: %s', package.misc_attributes)
+
         package.save()
-        LOGGER.info('Ingest ID: %s', external_id)
-        LOGGER.info('External ID: %s', external_id)
+
     elif status =='failed':
         LOGGER.error('Ingest failed')
         package.status = Package.FAIL
         package.save()
         for event in ingest['events']:
             LOGGER.info('{type}: {description}'.format(**event))
-    #else:
-    #    LOGGER.error('Unknown ingest status %s' % status)
-    #    package.status = Package.FAIL
-    #    package.save()
+
+    else:
+        LOGGER.info("Package status: %s", status)
```

Comment:
```suggestion
        LOGGER.warn("Unrecognised package status: %s", status)
```

Make it super obvious that something unexpected has happened.